ephemeral/database/index.js

```diff
@@ -0,0 +1,149 @@
+const core = require("@actions/core");
+const yaml = require("yaml");
+const axios = require("axios");
+
+const sleep = ms => new Promise(r => setTimeout(r, ms));
+
+const connectionInformation = database => {
+  if (database.status !== 'Running') {
+    throw new Error(`Database ${database.name} is not running, cannot get connection information`);
+  }
+
+  return {
+    databaseEntityId: database.databaseEntityId,
+    status: database.status,
+    host: `${database.kubernetesServiceName}.${database.kubernetesNamespace}`,
+    port: database.port,
+    databaseName: database.databaseName,
+    user: database.databaseUserName,
+    password: database.databasePassword,
```

Comment:
It looks like this is encrypted, potentially with a server wide key -- is there a way to reverse this?

Reply:
this should be the raw password, same response property we give to a user in the UI to be able to auth

Reply:
I think I've misunderstood something in that case. 👍